urllib3/util/hpkp.py

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
+import collections
+import time
+
+from urllib3.exceptions import HPKPError
```

Comment:
I wasn't confident that a relative import beginning with `..` would go well on Python 2.6, and didn't really want to break my flow to check. Will it?